virttest/test_setup.py

```diff
@@ -2005,26 +2005,28 @@ def _set_polkit_conf(self):
         except Exception as e:
             raise PolkitRulesSetupError("Set polkit rules file failed: %s", e)
 
-    def setup(self):
+    def setup(self, params):
```

Comment:
we would pass the `params` during object creation, so it is not right to again pass it during method invocation of the same class. let's initialize `self.params = params` in `__init__()` and use it.